doc/user/cluster_tls.md

```diff
@@ -85,3 +85,30 @@ $ kubectl create secret generic operator-etcd-client-tls --from-file=etcd-ca-crt
 ```
 
 Pass `operator-etcd-client-tls` to `operatorSecret` field.
+
+### Access secure cluster
+
+Since we have set this domain in SAN in etcd server's client cert:
+```
+example-client.default.svc.cluster.local
+```
+We are gonna use this to access etcd cluster.
+If you want to use different DNS name or IP, you need to add that when generating etcd server's client certs.
+
+We are reusing the same certs opeartor is using:
+```
+etcd-crt.pem
+etcd-key.pem
+etcd-ca-crt.pem
+```
+If you want to use different certs, the certs should be generated from and trusted by etcd server's client CA `client-ca-crt.pem`.
+
+Talk to secure cluster:
+```
+$ ETCDCTL_API=3 etcdctl --endpoints=https://example-client.default.svc.cluster.local:2379 \
+    --cert=etcd-crt.pem --key=etcd-key.pem --cacert=etcd-ca-crt.pem \
+    member list -w table
+```
+It should be within a pod inside k8s in order to access the service name.
```

Comment:
move this to line 91. we should mention this when we talk about the FQDN we plan to use.